SOPT-iOS/Projects/Core/Sources/Error/OPAPIError.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  OPAPIError.swift
+//  Core
+//
+//  Created by 김영인 on 2023/04/22.
+//  Copyright © 2023 SOPT-iOS. All rights reserved.
+//
+
+import Foundation
+
+public enum OPAPIError: LocalizedError {
+    case attendanceError(BaseEntity<Data>)
+    
+    public var errorDescription: String?{
```

Comment:
진짜 나 개깐깐징어같은데 여기 한 칸 안띄웠어 . . . 
```suggestion
    public var errorDescription: String? {
```
내가 미안해 . . .